pkg/monitor/write_job_run_data.go

```diff
@@ -98,3 +101,26 @@ func computeDisruptionData(eventIntervals monitorapi.Intervals) *BackendDisrupti
 
 	return ret
 }
+
+func WriteClusterData(artifactDir string, _ monitorapi.ResourcesMap, events monitorapi.Intervals, timeSuffix string) error {
+	return writeClusterData(filepath.Join(artifactDir, fmt.Sprintf("job-variants%s.json", timeSuffix)), CollectJobVariants())
```

Comment:
Nit, but what do you think about calling this file cluster-data instead of job-variants? Just for discoverability by others -- I think if I were browsing around in a GCS bucket I'd be more apt to click on a file named 'cluster-data' instead of 'job-variants'.